frontend/src/components/Date/Date.scss

```diff
@@ -0,0 +1,7 @@
+.date {
+    font-size: 2em;
+    font-family: 'Roboto Condensed', sans-serif;;
+    font-weight: 900;
+    color: #102A71;
+    margin: 0.5vh;
+}
```

Comment:
this selector is not specific `,date` has only 0,1,0. It may cause some problems in the future - every component with date className will have this styling assigned. 

Consider making it more specific (add chaining or multiple class names).